src/Symfony/Bundle/SecurityBundle/DependencyInjection/Security/AccessToken/OidcTokenHandlerFactory.php

```diff
@@ -42,7 +43,7 @@ public function create(ContainerBuilder $container, string $id, array|string $co
         if (\in_array($config['algorithm'], ['ES256', 'ES384', 'ES512'], true)) {
             $tokenHandlerDefinition->replaceArgument(0, new Reference('security.access_token_handler.oidc.signature.'.$config['algorithm']));
         } else {
-            $tokenHandlerDefinition->replaceArgument(0, (new ChildDefinition('security.access_token_handler.oidc.signature')))
+            $tokenHandlerDefinition->replaceArgument(0, new ChildDefinition('security.access_token_handler.oidc.signature'))
```

Comment:
wait, the closing bracket should be moved after replaceArgument, and the opening one be kept, isn't it?